sandpack-react/src/presets/Sandpack.tsx

```diff
@@ -194,6 +283,32 @@ const ConsoleCounterButton: React.FC<{
   );
 };
 
+const dragHandler = css({
+  position: "absolute",
+  zIndex: "$top",
+
+  variants: {
+    direction: {
+      vertical: {
+        right: 0,
+        left: 0,
+        height: 10,
+        cursor: "ns-resize",
+      },
+      horizontal: {
+        top: 0,
+        bottom: 0,
+        width: 10,
```

Comment:
did you increase this from the time you shared the demo the first time? I was mentioning that the interaction area with the drag handler was very small

Reply:
Yeah, I did. So the drag handler should be more accessible now